RaidExtractor.Core/AccountDumpClient.cs

```diff
@@ -452,7 +455,30 @@ public partial class Hero
         public List<int> Masteries { get; set; }
     }
 
-    [System.CodeDom.Compiler.GeneratedCode("NSwag", "13.7.0.0 (NJsonSchema v10.1.24.0 (Newtonsoft.Json v11.0.0.0))")]
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.1.24.0 (Newtonsoft.Json v11.0.0.0)")]
+    public partial class ShardSummonInfo
+    {
+        [Newtonsoft.Json.JsonProperty("rarity", Required = Newtonsoft.Json.Required.Always)]
+        public string Rarity { get; set; }
+
+        [Newtonsoft.Json.JsonProperty("pullCount", Required = Newtonsoft.Json.Required.Always)]
+        public int PullCount { get; set; }
+
+        [Newtonsoft.Json.JsonProperty("lastHeroId", Required = Newtonsoft.Json.Required.Always)]
+        public int LastHeroId { get; set; }
+    }
+
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.1.24.0 (Newtonsoft.Json v11.0.0.0)")]
+    public partial class ShardInfo
+    {
+        [Newtonsoft.Json.JsonProperty("count", Required = Newtonsoft.Json.Required.Always)]
+        public int Count { get; set; }
+
+        [Newtonsoft.Json.JsonProperty("summonData", Required = Newtonsoft.Json.Required.Always)]
+        public List<ShardSummonInfo> SummonData { get; set; }
+    }
+
+        [System.CodeDom.Compiler.GeneratedCode("NSwag", "13.7.0.0 (NJsonSchema v10.1.24.0 (Newtonsoft.Json v11.0.0.0))")]
```

Comment:
Indentation is a little off on line 481 here

Reply:
Arg! missed one, thanks.